server/sonar-web/src/main/js/components/common/BranchStatus.tsx

```diff
@@ -52,21 +55,33 @@ export default function BranchStatus({ branch, concise = false }: Props) {
       </ul>
     ) : (
       <ul className="branch-status">
-        <li className="spacer-right">
+        <li className="little-spacer-right">
           <StatusIndicator color={indicatorColor} size="small" />
         </li>
         <li className="spacer-left">
           {branch.status.bugs}
-          <BugIcon />
+          <BugIcon className="little-spacer-left" />
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3115&issues=AWIJ6cPiGvJAiVQc9UOv&open=AWIJ6cPiGvJAiVQc9UOv)

[ISSUE_KEY]: AWIJ6cPiGvJAiVQc9UOv